packages/relay/src/lib/eth.ts

```diff
@@ -945,6 +945,10 @@ export class EthImpl implements Eth {
     return input === null ? null : EthImpl.numberTo0x(input);
   }
 
+  static nullableNonceNumberTo0x(input: number | BigNumber): string | null {
```

Comment:
I think `nullable` in the name implies that it can be `null`. Also I'm not sure it's worth adding another function for that, instead we can do `EthImpl.numberTo0x(contractResultDetails.nonce || 0)`

Reply:
Good point.